app/Console/Commands/Node/MakeNodeCommand.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * Pterodactyl - Panel
+ * Copyright (c) 2015 - 2017 Dane Everitt <dane@daneeveritt.com>.
+ *
+ * This software is licensed under the terms of the MIT license.
+ * https://opensource.org/licenses/MIT
+ */
+
+namespace Pterodactyl\Console\Commands\Node;
+
+use Illuminate\Console\Command;
+use Pterodactyl\Services\Nodes\NodeCreationService;
+
+class MakeNodeCommand extends Command
+{
+    /**
+     * @var \Pterodactyl\Services\Nodes\NodeCreationService
+     */
+    protected $creationService;
+
+    /**
+     * @var string
+     */
+    protected $signature = 'p:node:make
+                            {--name= : A name to identify the node.}
+                            {--description= : A description to identify the node.}
+                            {--locationId= : A valid locationId.}
+                            {--fqdn= : The domain name (e.g node.example.com) to be used for connecting to the daemon. An IP address may only be used if you are not using SSL for this node.}
+                            {--public= : Should the node be public or private? (public=1 / private=0).}
+                            {--scheme= : Which scheme should be used? (Enable SSL=https / Disable SSL=http).}
+                            {--proxy= : Is the daemon behind the cloudflare proxy? (Yes=1 / No=0).}
+                            {--maintenance= : Should maintenance mode be enabled? (Enable Maintenance mode=1 / Disable Maintenance mode=0).}
+                            {--maxMemory= : Set the max memory amount.}
+                            {--overallocateMemory= : Enter the amount of ram to overallocate (% or -1 to overallocate the maximum).}
+                            {--maxDisk= : Set the max disk amount.}
+                            {--overallocateDisk= : Enter the amount of disk to overallocate (% or -1 to overallocate the maximum).}
+                            {--uploadSize= : Enter the maximum upload filesize.}
+                            {--daemonListeningPort= : Enter the wings listening port.}
+                            {--daemonSFTPPort= : Enter the wings SFTP listening port.}
+                            {--daemonBase= : Enter the base folder.}';
+
+    /**
+     * @var string
+     */
+    protected $description = 'Creates a new node on the system via the CLI.';
+
+    /**
+     * Create a new command instance.
+     */
+    public function __construct(NodeCreationService $creationService)
+    {
+        parent::__construct();
+
+        $this->creationService = $creationService;
+    }
+
+    /**
+     * Handle the command execution process.
+     *
+     * @throws \Pterodactyl\Exceptions\Model\DataValidationException
+     */
+    public function handle()
+    {
+        $name = $this->option('name') ?? $this->ask(trans('command/messages.node.ask_name'));
+        $description = $this->option('description') ?? $this->ask(trans('command/messages.node.ask_description'));
```

Comment:
You can remove all of the translation stuff for this, that was a bad choice made awhile back but it is pretty pointless for the CLI.